webpack.config.js

```diff
@@ -79,27 +81,33 @@ if (NPM_RUN === 'release') {
   configurations.push(webpackConfig({
     filename: 'dash-shaka-playback.min.js',
     optimization: {
-      minimizer: [
-        new UglifyJsPlugin({
-          sourceMap: true
-        }),
-      ]
+      minimizer: [new TerserPlugin({ 
+        extractComments: false,
+        terserOptions: {
+          format: {
+            comments: false,
+          },
+        },
```

Comment:
Why did you change `UglifyJsPlugin` to `TerserPlugin`?

Reply:
Webpack was complaining that the plugin had been deprecated, and Webpack itself suggested TerserPlugin as an option.

UglifyJsPlugin has been archived by the owner on Dec 5, 2019.